README.md

```diff
@@ -11,6 +11,11 @@ The package can be installed using pip:
 ```
 $ pip3 install elicznik
 ```
+or optionally:
```

Comment:
Let's phrase it *or alternatively:*.  

This way it's clear that it's not an optional extra step, but an alternative to the one above.